FewCore/Component.swift

```diff
@@ -234,15 +234,19 @@ public class Component<S>: Element {
 	}
 
 	final private func enqueueRender() {
-		if renderQueued { return }
-
-		renderQueued = true
-
-		let observer = CFRunLoopObserverCreateWithHandler(kCFAllocatorDefault, CFRunLoopActivity.Exit.rawValue, 0, 0) { _, _ in
-			self.renderQueued = false
-			self.render()
-		}
-		CFRunLoopAddObserver(CFRunLoopGetMain(), observer, kCFRunLoopCommonModes)
+#if os(iOS)
```

Comment:
Because the enqueueing didn't work on iOS, but i understand that we want to enqueue this on the end on the main run loop :sweat_smile:. I'll fix this with `NSOperationQueue`.